android/app/build.gradle

```diff
@@ -90,7 +90,7 @@ import com.android.build.OutputFile
  */
 
 project.ext.react = [
-    enableHermes: false,  // clean and rebuild if changing
+    enableHermes:true,  // clean and rebuild if changing
```

Comment:
Nice I think this is supposed to be better for Android?

Reply:
Yeah, it's a JS engine optimized for Android. It is required in order to use Reanimated2. We'll see...